src/Screen/Fields/Relation.php

```diff
@@ -92,7 +92,12 @@ class Relation extends Field
      */
     public function fromModel(string $model, string $name, string $key = null): self
```

Comment:
This is old PR and I forgot to pull BEFORE pushing a new feature 😅

Reply:
Please do a rebase to avoid this.